web/src/repo/TreePage.tsx

```diff
@@ -274,20 +272,13 @@ export class TreePage extends React.PureComponent<Props, State> {
                                 </header>
                             )}
                             <section className="tree-page__section">
-                                <h3 className="tree-page__section-header">
+                                <Link
+                                    className="btn btn-primary d-inline-flex align-items-center"
+                                    to={`/search?${buildSearchURLQuery(this.getQueryPrefix())}`}
+                                >
+                                    <SearchIcon className="icon-inline"></SearchIcon>
                                     Search in this {this.props.filePath ? 'tree' : 'repository'}
```

Comment:
This is consistent with our "Files and directories" labels below on this page

```suggestion
                                    Search in this {this.props.filePath ? 'directory' : 'repository'}
```